csrc/host_ir/container.cpp

```diff
@@ -54,6 +55,11 @@ void HostIrContainer::setKernelExecutor(
   kernel_executors_.at(index) = std::move(ke);
 }
 
+void HostIrContainer::sortExprs()
+{
+  this->top_level_exprs_ = groupExprPrintSorting(this->top_level_exprs_);
```

Comment:
Did you try https://github.com/NVIDIA/Fuser/blob/4d2db45698435920a90036c4bf1912a94808b6eb/csrc/iter_visitor.cpp#L904 or https://github.com/NVIDIA/Fuser/blob/4d2db45698435920a90036c4bf1912a94808b6eb/csrc/iter_visitor.cpp#L892? groupExprPrintSorting hasn't been optimized for speed. It's fine to use it for debugging but I'd be careful with using it elsewhere